### PR TITLE
feat(elements|ino-snackbar): rework snackbar

### PR DESCRIPTION
**Pull request by janivo**
## Proposed Changes

- Types
	- remove `primary` and `warning` ‼️
 	- add `info` and `success`
	- set `info` as default
	- update design
- Alignment
	- remove `alignment` property ‼️
	- add css vars to make position customizable (`ino-snackbar-(top|right|bottom|left)`) (can be useful to implement stacking)
	- set top-right as the default position
- Design
	- add new designs for `info` and `success`
	- update action-button, shapes and layout according to the design
	- move the close-Button to the top-right edge of the notification window

### CODE REVIEW

File: packages/elements/src/components/ino-snackbar/ino-snackbar.scss [hunk 1]
```diff
@@ -4,89 +4,125 @@
...
+    box-shadow-color: #2d5dff29,
+  ),
+  error: (
+    color: #7df481,
+    box-shadow-color: #7df48129,
+  ),
+  success: (
+    color: #f29191,
+    box-shadow-color: #f2919129,
+  ),
```
Comment (BenPag) [inline packages/elements/src/components/ino-snackbar/ino-snackbar.scss:21]: I think the colors should be swaped. `success` is green and `error` is red.

File: packages/elements/src/components/ino-snackbar/ino-snackbar.scss [hunk 2]
```diff
@@ -4,89 +4,125 @@
...
+);
```
Comment (BenPag) [inline packages/elements/src/components/ino-snackbar/ino-snackbar.scss:22]: These colors are the new elements colors I guess. I think it is better to use the colors defined in `theme.color` or at least mention it in the issue that hanfles the implementation of the new color pallet.

**Review by BenPag — CHANGES_REQUESTED** [review]
Great Job! Looks very nice 👍

File: packages/elements/src/components/ino-snackbar/ino-snackbar.scss (hunk 2 above)
Comment (janivo) [reply packages/elements/src/components/ino-snackbar/ino-snackbar.scss]: As we discussed earlier, I switched to the old color pallette for now.

**Comment by janivo** [comment]
blocked: waiting for the new icons

**Pull request merged**